tests/ignite/contrib/metrics/regression/test_maximum_absolute_error.py

```diff
@@ -60,3 +61,45 @@ def test_maximum_absolute_error():
     np_max = np.max(np.abs((d - ground_truth)))
     np_ans = np_max if np_max > np_ans else np_ans
     assert m.compute() == pytest.approx(np_ans)
+
+
+def test_integration():
+    def _test(y_pred, y, batch_size):
+        def update_fn(engine, batch):
+            idx = (engine.state.iteration - 1) * batch_size
+            y_true_batch = np_y[idx : idx + batch_size]
+            y_pred_batch = np_y_pred[idx : idx + batch_size]
+            return idx, torch.from_numpy(y_pred_batch), torch.from_numpy(y_true_batch)
+
+        engine = Engine(update_fn)
+
+        m = MaximumAbsoluteError(output_transform=lambda x: (x[1], x[2]))
+        m.attach(engine, "mae")
+
+        np_y = y.numpy()
+        np_y_pred = y_pred.numpy()
+
+        data = list(range(y_pred.shape[0] // batch_size))
+        mae = engine.run(data, max_epochs=1).metrics["mae"]
+
+        np_ans = -1
+
+        np_max = np.max(np.abs((np_y_pred - np_y)))
+        np_ans = np_max if np_max > np_ans else np_ans
+
+        assert np_ans == pytest.approx(mae)
```

Comment:
can't we compare directly with `np_max` ?

Reply:
If you mean like that:
```
        np_max = np.max(np.abs((np_y_pred - np_y)))
        np_max = np_max if np_max > -1 else -1            
        assert np_max == pytest.approx(mae)
```
Yes this can be done.

Reply:
No, I mean why we have to compare it to -1 ?

Reply:
Yes this can be done, but i wanted to follow the same way the metrics implemented [here](https://github.com/pytorch/ignite/blob/master/ignite/contrib/metrics/regression/maximum_absolute_error.py)

Reply:
Well, in which case you can have MAE equals -1 :) ?